src/ray/raylet/scheduling/fixed_point.h

```diff
@@ -27,16 +27,14 @@ class FixedPoint {
 
  public:
   FixedPoint() : FixedPoint(0.0) {}
-  FixedPoint(double d) { i_ = (uint64_t)(d * RESOURCE_UNIT_SCALING); }  // NOLINT
+  FixedPoint(double d) { i_ = (int64_t)(d * RESOURCE_UNIT_SCALING); }  // NOLINT
 
   FixedPoint(int i) { i_ = (i * RESOURCE_UNIT_SCALING); }  // NOLINT
 
   FixedPoint(uint32_t i) { i_ = (i * RESOURCE_UNIT_SCALING); }  // NOLINT
 
   FixedPoint(int64_t i) : FixedPoint((double)i) {}  // NOLINT
```

Comment:
actually, any reason why can't we just do:
```
  FixedPoint() : FixedPoint(0.0) {}
  FixedPoint(double d) { i_ = static_cast<int64_t>(d * RESOURCE_UNIT_SCALING); }  // NOLINT
  FixedPoint(int64_t i) : { i_ = i * RESOURCE_UNIT_SCALING; }  
```

cc @iycheng @jjyao

Reply:
Seems fine to me.

Reply:
So, need I remove the two constructors in this PR? cc @scv119 
```
FixedPoint(int i) { i_ = (i * RESOURCE_UNIT_SCALING); }  // NOLINT
FixedPoint(uint32_t i) { i_ = (i * RESOURCE_UNIT_SCALING); }  // NOLINT
```

Reply:
Hi, @scv119 
I decide to only remove `FixedPoint(uint32_t i) ` for the time being in this PR, because there are many compile error if I remove `FixedPoint(int i) ` too.